docs/readmes/orc8r/upgrade_1_4.md

```diff
@@ -0,0 +1,66 @@
+---
+id: upgrade_1_4
+title: Upgrade to v1.4
+hide_title: true
+---
+
+# Upgrade to v1.4
+
+This guide covers upgrading Orchestrator deployments from v1.3 to v1.4.
+Upgrades which skip versions (e.g. v1.2 to v1.4) are not explicitly documented
+or supported at this time.
+
+The v1.4 upgrade follows a standard procedure:
+
+- Upgrade the Terraform modules
+- Bump version numbers
+
+## Prerequisites
+
+Build and publish the application containers and helm charts on the head of the
+release branch by following the documentation [here](https://docs.magmacore.org/docs/orc8r/deploy_build).
```

Comment:
"release branch by following the [Build Orchestrator](http://example.com) documentation."

Ref: http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/